modules/library/src/main/scala/zio/elasticsearch/highlighting/Highlights.scala

```diff
@@ -0,0 +1,27 @@
+package zio.elasticsearch.highlighting
+
+import zio.Chunk
+import zio.elasticsearch.highlighting.Highlights.HighlightConfig
+import zio.json.ast.Json
+import zio.json.ast.Json.Obj
+
+case class Highlights(fields: Chunk[HighlightField], globalConfig: HighlightConfig = Map.empty) { self =>
```

Comment:
Can we use `config` here instead of `globalConfig`?

Reply:
We have config parameter in method, maybe it would not be clear.